src/sessions.ts

```diff
@@ -96,6 +103,14 @@ class ClientSession extends TypedEventEmitter<ClientSessionEvents> {
   transaction: Transaction;
   /** @internal */
   [kServerSession]?: ServerSession;
+  /** @internal */
+  [kSnapshotTime]?: Timestamp;
+  /** @internal */
+  [kSnapshotEnabled] = false;
+
+  get snapshotEnabled(): boolean {
```

Comment:
suggestion: `isSnapshotEnabled`?
Can you move it be next to the other getters?
I'm in favor of the helper though regardless of these two asks

Reply:
Moved the getter (as discussed, leaving name unchanged so that the verb doesn't mislead anyone into thinking this is a function call)